README.rst

```diff
@@ -148,10 +149,11 @@ If no specific bucket name is provided, the "default" bucket is used.
     client.update_bucket(id='payments', data={'description': 'My payments data.'})
 
     # Or modify some fields in an existing bucket.
-    client.patch_bucket(id='payments', data={'status': 'updated'})
+    # The Kinto server supports different types of patches, which can be used from kinto_http.patch_type.
+    client.patch_bucket(id='payments', data=BasicPatch({'status': 'updated'}))
 
     # It is also possible to manipulate bucket permissions (see later)
-    client.patch_bucket(id='payments', permissions={})
+    client.patch_bucket(id='payments', data=BasicPatch({'status': 'updated'}), permissions={})
```

Comment:
I agree, if there is no test then it does not exist.

However, I believe this is a relevant use-case.

You can remove this from the README and open an issue to support it.